src/pyodide/internal/python.js

```diff
@@ -419,20 +419,27 @@ function setUploadFunction(toUpload) {
   };
 }
 
+// "\x00snp"
+const SNAPSHOT_MAGIC = 0x6d736100;
```

Comment:
Maybe instead of using the wasm magic constant we should use something random?

```
$ capnp id
@0xa6a71b7b00626e55
```

You can truncate the above.

Reply:
This isn't the wasm magic number, that's `"\x00asm"` this is `"\x00snp"`. Things that are not valid webassembly should not start with the wasm magic number. I think there is an advantage to having the magic number be something one can actually read. But either way is probably fine.

Reply:
Hm, are you sure? I see it here: https://chromium.googlesource.com/v8/v8/+/refs/heads/main/src/wasm/wasm-constants.h#22

Reply:
Fixed. Thanks!